frontend/src/app/trees/admin/report/results/report-results.component.ts

```diff
@@ -31,6 +31,18 @@ export class ReportResultsComponent implements OnChanges {
       showTitle: false,
       useBom: false
     };
-    const ng2csv = new Angular2Csv(this.result.permits, 'Christmas Trees Permits Report', options);
+
+    const orderedPermits = [];
```

Comment:
seems like an ORDER BY on the backend would be better?  Also we should sort the array itself by date asc I think